Improvement/snippet formatting

SufianBabri opened the pull request:
Some snippets weren't showing with the right formatting on the website, I've fixed their formatting.

Few of changes improve the readability immensely, while others are mostly minor/cosmetic improvements.

Following is the preview of some of the changes that are made (before -> now):

![old-vs-new](https://github.com/user-attachments/assets/13fed5e4-e89a-48c7-baf8-ea6beeb353d3)

SufianBabri committed:
Fix the formatting of Animated Header component

SufianBabri committed:
Fix the formatting of 3D Button component

SufianBabri committed:
Fix the formatting of Pulse Button component

SufianBabri committed:
Fix the formatting of Shadow Button component

SufianBabri committed:
Fix formatting of floating text input, gradient button and step button components

SufianBabri committed:
Fix the formatting of scale button component

SufianBabri commented:
I did it manually.

waleedcj reviewed (commented):
please add the last changed data

SufianBabri commented:
I have updated the `lastModified` of each of the files I've changed.

SufianBabri commented:
@waleedcj Hi. I've updated the timestamps against the `lastModified` property in all the files that were changed (I've rewritten the git history so that everything looks clean when the PR gets merged).
